lib/OnboardingMessageProvider.jsm

```diff
@@ -223,7 +224,10 @@ const ONBOARDING_MESSAGES = async () => ([
       icon: "tracking",
       primary_button: {
         label: {string_id: "onboarding-tracking-protection-button2"},
-        action: {
+        action: Services.locale.appLocaleAsLangTag.substr(0, 2) === "en" ? {
+          type: "OPEN_URL",
+          data: {args: "https://mzl.la/ETPdefault", where: "tabshifted"},
```

Comment:
I guess this will be published that day.

Reply:
Is there some documentation for this URL somewhere?

Reply:
https://bugzilla.mozilla.org/show_bug.cgi?id=1550858#c7